provider/vsphere/environ_broker_test.go

```diff
@@ -372,7 +372,7 @@ func (s *legacyEnvironBrokerSuite) TestStartInstanceDatastoreDefault(c *gc.C) {
 	_, err = s.env.StartInstance(s.callCtx, s.createStartInstanceArgs(c))
 	c.Assert(err, jc.ErrorIsNil)
 
-	call := s.client.Calls()[3]
+	call := s.client.Calls()[4]
```

Comment:
Something feels a bit wrong about these tests that you would tie it that tightly to the exact call. I don't think it is on you to fix it now, but it does feel a little fragile as a test.

Reply:
I know -- I found those tests very fragile for that reason. I don't think this change is big enough to make it worth refactoring this now though.